db/helpers.go

```diff
@@ -180,3 +215,34 @@ func dynamicUpdateDocument(item interface{}, alwaysUpdateTags []string) (bson.M,
 	}
 	return bson.M{"$set": update}, nil
 }
+
+// readSubscriptionJSON reads a JSON file with an array of subscritpions
+// and return it as a Subscription array
+func readSubscriptionJSON(subscriptionsFile string) ([]*Subscription, error) {
+	if subscriptionsFile == "" {
+		subscriptionsFile = "subscriptions.json"
```

Comment:
this probably should be a constant

Reply:
and probably should be configurable via envvars or cli args

Reply:
@lucasmenendez  can you review that the weay the env var is handled is OK with you?